micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/SidebarAddEditItems.js

```
@@ -0,0 +1,537 @@
+            />}
+            {showPopup && <PopUp
+                // className="localisation-popup-container"
+                heading={t("ADD_TRANSLATIONS")}
+                onClose={() => setShowPopup(false)}
+            >
+                <div className="localisation-inputs">
+                    {locales.map((locale) => (
+                        <div key={locale} className="localisation-row">
+                            <label className="locale-label">
+                                {t(locale)}
+                            </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)